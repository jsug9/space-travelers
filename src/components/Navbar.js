import React from 'react';
import { Link } from 'react-router-dom';

function Navbar() {
  return (
    <nav>
      <h3>Space Travelers</h3>
      <ul>
        <Link to="/">Rockets</Link>
        <Link to="/missions">Missions  | </Link>
        <Link to="/profile">My Profile</Link>
      </ul>
    </nav>
  );
}

export default Navbar;