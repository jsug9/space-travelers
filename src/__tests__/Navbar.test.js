import { BrowserRouter } from 'react-router-dom';
import TestRenderer from 'react-test-renderer';
import Navbar from '../components/Navbar';

it('Check if Navbar renders correctly', () => {
  const tree = TestRenderer.create(
    <BrowserRouter>
      <Navbar />
    </BrowserRouter>,
  );
  expect(tree).toMatchSnapshot();
});
