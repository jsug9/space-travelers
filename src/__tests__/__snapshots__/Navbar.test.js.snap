// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check Navbar`] = `
<nav>
  <div
    className="rightNav"
  >
    <img
      alt="logo"
      className="logo"
      src="planet.png"
    />
    <h1>
      Space Travelers' Hub
    </h1>
  </div>
  <ul
    className="links"
  >
    <li
      className="linkLi"
    >
      <a
        aria-current="page"
        className="active"
        href="/"
        onClick={[Function]}
      >
        Rockets
      </a>
    </li>
    <li
      className="linkLi"
    >
      <a
        className="link"
        href="/missions"
        onClick={[Function]}
      >
        Missions
      </a>
    </li>
    <li
      className="linkLi"
    >
      <a
        className="link"
        href="/profile"
        onClick={[Function]}
      >
        My Profile
      </a>
    </li>
  </ul>
</nav>
`;

exports[`Check if Navbar renders correctly 1`] = `
<nav>
  <div
    className="rightNav"
  >
    <img
      alt="logo"
      className="logo"
      src="planet.png"
    />
    <h1>
      Space Travelers' Hub
    </h1>
  </div>
  <ul
    className="links"
  >
    <li
      className="linkLi"
    >
      <a
        aria-current="page"
        className="active"
        href="/"
        onClick={[Function]}
      >
        Rockets
      </a>
    </li>
    <li
      className="linkLi"
    >
      <a
        className="link"
        href="/missions"
        onClick={[Function]}
      >
        Missions
      </a>
    </li>
    <li
      className="linkLi"
    >
      <a
        className="link"
        href="/profile"
        onClick={[Function]}
      >
        My Profile
      </a>
    </li>
  </ul>
</nav>
`;
